_pages/en_US/installing-boot9strap-(slottool).txt

```diff
@@ -0,0 +1,94 @@
+---
+title: "Installing boot9strap (slotTool)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This method of using Seedminer for further exploitation uses your `movable.sed` file to gain access to the Homebrew Launcher using the PicHaxx exploit for the purposes of injecting an exploited WiFi profile.
+
+Once the WiFi profile has been injected, we will use SAFE_MODE, which is a recovery feature present on all 3DS consoles, to activate the exploited WiFi profile.
+
+If your device already has access to the Homebrew Launcher (whether through PicHaxx or another exploit), replace your current otherapp with an otherapp for your console's region/revision from the [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest) release `.zip` located in the `otherapps_with_CfgS` folder.
+  + Since you have access to the Homebrew Launcher you may skip to section III of this page 
```

Comment:
This text might be a bit confusing and may be unnecessary depending on what other changes are made to this page